ppr-ui/src/components/tables/SearchHistory.vue

```diff
@@ -58,29 +59,38 @@
                     </v-col>
                   </v-row>
                 </td>
-                <td v-if="isPprSearch(item)">
-                  {{ displayType(item.searchQuery.type) }}
-                </td>
-                <td v-else>
-                  {{ displayMhrType(item.searchQuery.type) }}
+                <td>
+                  <span class="aria-label-only"
+                    aria-hidden="false"
+                  >{{ headers[1].text }}</span>
+                  {{ isPprSearch(item) ? displayType(item.searchQuery.type) : displayMhrType(item.searchQuery.type) }}
                 </td>
                 <td>
+                  <span class="aria-label-only"
+                    aria-hidden="false"
+                  >{{ headers[2].text }}</span>
                   <span v-if="isPprSearch(item)">Personal Property</span>
                   <span v-else>Manufactured Homes</span>
                 </td>
                 <td>
+                  <span class="aria-label-only"
+                    aria-hidden="false"
+                  >{{ headers[3].text }}</span>
```

Comment:
Some spacing and tabbing to fix up here. 
Also, just want to double check the span doesn't have an impact on styling here?